src/test/regress/expected/subquery_basics.out

```diff
@@ -192,7 +192,7 @@ DEBUG:  Plan XXX query after replacing subqueries and CTEs: SELECT foo.value_2,
 
 -- same query with alias in the subquery
 SELECT
-   DISTINCT ON (citus) citus, postgres, citus + 1 as c1, postgres-1 as p1
+   DISTINCT ON (citus) citus, <user>, citus + 1 as c1, <user>-1 as p1
```

Comment:
Here it seems we shouldn't have normalized. Maybe just rename these in the sql file so they don't get normalized.